Sources/SourceKitD/SourceKitD.swift

```diff
@@ -107,9 +129,7 @@ extension SourceKitD {
 }
 
 private func logRequest(_ request: SKDRequestDictionary) {
-  // FIXME: Ideally we could log the request key here at the info level but the dictionary is
-  // readonly.
-  logger.log(
+  logger.info(
```

Comment:
What's default vs info? The docs weren't particularly illuminating.

Reply:
https://github.com/apple/sourcekit-lsp/blob/78c28f5dec294ffa87ffcbd91fb9bbeae9d8cbd4/Sources/LSPLogging/Logging.swift#L13-L18

Reply:
Ah, so persisted vs not 👍